modules/storage-expanding-csi-volumes.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies
+//
+// * storage/expanding-persistent-volumes.adoc
+
+[id="expanding-csi-volumes_{context}"]
+= Expanding CSI volumes
+
+The Container Storage Interface (CSI) allows {product-title} to expand storage volumes after they have already been created.
```

Comment:
ISG warns about "allows" in this context in appendix C, but it's not a showstopper.

Reply:
Glad you pointed out the ref. This copy was borrowed from existing content, but I agree with ISG's suggestion to rewrite more directly. It's not a permissions thing, either, so I rewrote this and made small changes to paragraph ordering for clarity and readability. Merging.